README.md

```diff
@@ -251,7 +250,7 @@ tests, and ensure that `npm test` continues to pass.
 
 ## License
 
-MIT License. © 2012 Aseem Kishore, and [others](
+MIT License © 2012-2014 Aseem Kishore, and [others](
```

Comment:
Man, this is my fault since I missed your PR, but heck, may as well make this 2015 now. =D